docs/docs/command-line-interface.mdx

```diff
@@ -416,6 +417,43 @@ domain files:
 ```text [rasa data validate --help]
 ```
 
+## rasa data augment nlu
```

Comment:
@akelad as discussed on Slack, it would be great if you could review the changes to 
* `docs/docs/command-line-interface.mdx` and
* `docs/docs/generating-nlu-data.mdx`

Reply:
How about using this one which also points to the forum? https://github.com/RasaHQ/rasa/blob/55a973d617dc5ffc79fb04722e9e43555178c40b/docs/docs/training-data-importers.mdx#L61